lms/urls.py

```diff
@@ -91,6 +91,8 @@
     # independently of courseware. https://github.com/edx/edx-val
     url(r'^api/val/v0/', include('edxval.urls')),
 
+    # Full Course/Library Import/Export API
+    url(r'^api/import_export/v1/', include('openedx.core.djangoapps.import_export.urls')),
```

Comment:
So Cale reminded me that the write restrictions happen at the field data layer and that this is reaching straight to the modulestore. In which case I think this is fine, but please hide it behind a feature flag and have the default behavior be that it does not appear in the LMS.